source/support.tex

```diff
@@ -664,6 +664,7 @@
 #define @\defnlibxname{cpp_lib_fstream_native_handle}@             202306L // also in \libheader{fstream}
 #define @\defnlibxname{cpp_lib_function_ref}@                      202306L // also in \libheader{functional}
 #define @\defnlibxname{cpp_lib_gcd_lcm}@                           201606L // also in \libheader{numeric}
+#define @\defnlibxname{cpp_lib_generate_random}@                   202403  // also in \libheader{random}
```

Comment:
On the LWG reflector, I raised the following issue:

> [P1068R11](https://wiki.edg.com/pub/Wg21tokyo2024/StrawPolls/P1068R11.pdf) adds `ranges::generate_random` *and nothing else*, with a feature-test macro:
>
> `__cpp_lib_generate_random`
>
> From the paper's changelog, it appears that it originally proposed more, and was eventually revised to provide only a ranges algorithm. It looks like the feature-test macro might not have been reconsidered. Should it instead follow the "ranges_meow" pattern as in:
>
> `__cpp_lib_ranges_generate_random`
>
> Precedents:
> * [P1223R5](https://wg21.link/P1223R5) was accepted with `__cpp_lib_find_last` but `__cpp_lib_ranges_find_last` was eventually applied by [LWG-3807](https://wg21.link/lwg3807).
> * [P2322R6](https://wg21.link/P2322R6) was accepted with `__cpp_­lib_fold` but `__cpp_lib_ranges_fold` was applied by #5645.
>
> Sorry for noticing this late - I know revising feature-test macros during the editing process isn't great, but I figured I'd mention it now that I noticed it.

@brevzin and @jwakely agreed with this rationale.

Reply:
adjusted